src/mono/mono/sgen/sgen-descriptor.c

```diff
@@ -129,7 +129,7 @@ mono_gc_make_descr_for_object (gsize *bitmap, int numbits, size_t obj_size)
 	}
 
 	if (first_set < 0) {
-		SGEN_LOG (6, "Ptrfree descriptor %p, size: %ld", (void*)desc, (long)stored_size);
+		SGEN_LOG (6, "Ptrfree descriptor %llu, size: %zu", (guint64)desc, stored_size);
```

Comment:
```suggestion
		SGEN_LOG (6, "Ptrfree descriptor %" PRIu64 ", size: %zu", (uint64_t)desc, stored_size);
```
`"%llu"` isn't portable

Reply:
It's technically undefined, but isn't it fine everywhere with C99? Happy to fix, but I don't think it actually matters.

Reply:
I don't think so.  Pretty sure it won't work on msvc.

Reply:
Its pretty hard to print out a guint64, using %p and GUINT_TO_POINTER () might fix it.

Reply:
Under VS2019 %llu works:

printf ("Max uint64_t: %llu", UINT64_MAX);

Max uint64_t: 18446744073709551615

and PRIu64 is defined to llu on msvc as well.

#define PRIu64       "llu"

but I guess including inttypes.h and use %" PRIu64 " is the more portable alternative.